doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1200,7 +1200,10 @@ Interactive debugger
 
 .. flag:: Ltac Debug
 
-   This option governs the step-by-step debugger that comes with the |Ltac| interpreter
+.. flag:: Debug Ltac
+
+   Two names for the same option, which governs the step-by-step
+   debugger that comes with the |Ltac| interpreter
```

Comment:
One of these two names should probably get deprecated while the other is kept and documented. Given that `Ltac Debug` was documented, I'd favor deprecating `Debug Ltac`.